metal3-dev/local-cbo.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash -xe
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
+
+LOGDIR=${SCRIPTDIR}/logs
+source $SCRIPTDIR/logging.sh
+
+source $SCRIPTDIR/common.sh
+source $SCRIPTDIR/utils.sh
+
+if ! which kubebuilder 2>&1 >/dev/null ; then
+    echo "Did not find kubebuilder." 1>&2
+    echo "To install it see https://book.kubebuilder.io/quick-start.html." 1>&2
+    exit 1
+fi
+
+if ! which yq 2>&1 >/dev/null ; then
+    echo "Did not find yq" 1>&2
+    echo "Install with: pip3 install --user yq" 1>&2
+    exit 1
+fi
+
+cbo_path=$GOPATH/src/github.com/openshift/cluster-baremetal-operator
+if [ ! -d $cbo_path ]; then
+    echo "Did not find $cbo_path" 1>&2
+    exit 1
+fi
+
+# Stop the machine-api-operator so it does not try to fix the
+# deployment we are going to change. This is required until MAO
+# will continue to check the presence of the CBO operator - in
+# addition to the annotations
+$SCRIPTDIR/stop-mao.sh
```

Comment:
If MAO is not stopped during a local CBO development then it's not clear how to bypass this check: https://github.com/openshift/machine-api-operator/blob/f38acc8d312f527cb2cf1505ef8e346a99a9638b/pkg/operator/sync.go#L208

Reply:
It's the responsibility of the CBO to configure its ClusterOperator resource and to annotate the metal3 deployment to tell the MAO to ignore it. Those two stories should be early in the list of implementation priorities, and until they are implemented it doesn't matter that the MAO keeps doing its work because the CBO doesn't do anything, yet.